src/databricks/labs/ucx/account.py

```diff
@@ -69,6 +71,77 @@ def sync_workspace_info(self):
             for installation in Installation.existing(ws, "ucx"):
                 installation.save(workspaces, filename=self.SYNC_FILE_NAME)
 
+    def create_account_level_groups(self):
+        acc_groups = self._get_account_groups()
+        all_valid_workspace_groups = self._get_valid_workspaces_groups()
+
+        for group_name, valid_group in all_valid_workspace_groups.items():
+            if group_name in acc_groups:
+                logger.info(f"Group {group_name} already exist in the account, ignoring")
+            else:
+                acc_group = self._ac.groups.create(display_name=group_name)
+
+                if len(acc_group.members) > 1:
+                    self._add_members_to_acc_group(acc_group.id, group_name, valid_group)
+
+                logger.info(f"Group {group_name} created in the account")
+
+    def _add_members_to_acc_group(self, acc_group_id: str, group_name: str, valid_group: Group):
+        for chunk in self._chunks(valid_group.members, 20):
+            logger.debug(f"Adding 20 members to acc group {group_name}")
+            self._ac.groups.patch(
+                acc_group_id,
+                operations=[iam.Patch(op=iam.PatchOp.ADD, path="members", value=[x.as_dict() for x in chunk])],
```

Comment:
Would love to see a demo of this 😉

Reply:
What do you mean there ? This code actually comes from test_performance.py